pkg/controller/bundle/bundle_unpacker_test.go

```diff
@@ -751,6 +756,523 @@ func TestConfigMapUnpacker(t *testing.T) {
+											},
+										},
+									},
+									Volumes: []corev1.Volume{
+										{
+											Name: "bundle",
+											VolumeSource: corev1.VolumeSource{
+												EmptyDir: &corev1.EmptyDirVolumeSource{},
+											},
+										},
+										{
+											Name: "util",
+											VolumeSource: corev1.VolumeSource{
+												EmptyDir: &corev1.EmptyDirVolumeSource{},
+											},
+										},
+									},
+								},
+							},
+						},
+						Status: batchv1.JobStatus{
+							Failed: 1,
+							Conditions: []batchv1.JobCondition{
+								{
+
+									Type:    batchv1.JobFailed,
+									Status:  corev1.ConditionTrue,
+									Message: "Job was active longer than specified deadline",
+									Reason:  "DeadlineExceeded",
+								},
+							},
+						},
+					},
+					&corev1.ConfigMap{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      pathHash,
+							Namespace: "ns-a",
+							OwnerReferences: []metav1.OwnerReference{
+								{
+									APIVersion:         "operators.coreos.com/v1alpha1",
+									Kind:               "CatalogSource",
+									Name:               "src-a",
+									Controller:         &blockOwnerDeletion,
+									BlockOwnerDeletion: &blockOwnerDeletion,
+								},
+							},
+						},
+					},
+				},
+				crs: []runtime.Object{
+					&operatorsv1alpha1.CatalogSource{
+						ObjectMeta: metav1.ObjectMeta{
+							Namespace: "ns-a",
+							Name:      "src-a",
+						},
+					},
+				},
+			},
+			args: args{
+				lookup: &operatorsv1alpha1.BundleLookup{
+					Path:     bundlePath,
+					Replaces: "",
+					CatalogSourceRef: &corev1.ObjectReference{
+						Namespace: "ns-a",
+						Name:      "src-a",
+					},
+					Conditions: []operatorsv1alpha1.BundleLookupCondition{
+						{
+							Type:               operatorsv1alpha1.BundleLookupPending,
+							Status:             corev1.ConditionTrue,
+							Reason:             JobIncompleteReason,
+							Message:            JobIncompleteMessage,
+							LastTransitionTime: &start,
+						},
+					},
+				},
+			},
+			expected: expected{
+				res: &BundleUnpackResult{
+					name: pathHash,
+					BundleLookup: &operatorsv1alpha1.BundleLookup{
+						Path:     bundlePath,
+						Replaces: "",
+						CatalogSourceRef: &corev1.ObjectReference{
+							Namespace: "ns-a",
+							Name:      "src-a",
+						},
+						Conditions: []operatorsv1alpha1.BundleLookupCondition{
+							{
+								Type:               operatorsv1alpha1.BundleLookupPending,
+								Status:             corev1.ConditionTrue,
+								Reason:             JobIncompleteReason,
+								Message:            JobIncompleteMessage,
+								LastTransitionTime: &start,
+							},
+							{
+								Type:               BundleLookupFailed,
+								Status:             corev1.ConditionTrue,
+								Reason:             "DeadlineExceeded",
+								Message:            "Job was active longer than specified deadline",
+								LastTransitionTime: &start,
+							},
+						},
```

Comment:
Now that I'm looking at it directly, I could see how having both `Pending` and `Failed` present and `True` can be confusing for users.